src/paid/handler.js

```diff
@@ -132,22 +183,33 @@ function enrichPageTypes(auditResult, classifier) {
   return enriched;
 }
 
-function tryEnrich(segments, classifier, auditUrl) {
+function tryEnrich(segments, classifier, auditUrl, log) {
   if (isNonEmptyObject(classifier)) {
     return enrichPageTypes(segments, classifier);
   }
-  log.warn(`Page type configuration is missing for site ${auditUrl}. Proceeding with auit without page and url enrichment`);
-
+  log.warn(`[paid-audit] [Site: ${auditUrl}] Page type configuration missing, proceeding without enrichment`);
   return segments;
 }
 
-export async function paidAuditRunner(auditUrl, context, site) {
-  if (typeof log === 'undefined') {
-    log = context.log;
-  }
+function buildMystiqueMessage(site, audit, url) {
+  return {
+    type: AUDIT_CONSTANTS.TYPES.MYSTIQUE,
+    observation: AUDIT_CONSTANTS.OBSERVATION,
+    siteId: site.getId(),
+    url,
+    auditId: audit.getAuditId(),
```

Comment:
if the `audit` object here is the audit entity from dynamodb, then please make sure `getAuditId` exists. Afaik, it should be `getId`, not `getAuditId`. Can you make sure that you are setting auditId field to a value with a log?